src/shadow-embed.js

```diff
@@ -219,7 +231,7 @@ export function importShadowBody(shadowRoot, body, deep) {
  * @return {string}
  */
 export function transformShadowCss(shadowRoot, css) {
-  if (isShadowDomSupported()) {
+  if (isShadowCssSupported()) {
```

Comment:
Looks a bit duplicate - we only install transformer when `isShadowCssSupported() == true`, right?

Reply:
**tl;dr** Yes it was duplicate, done :)

Ah yes, there was a sort of duplicate. I was calling `transformShadowCss` in both `createShadowRootPolyfill` and `createShadowRoot`. Where `createShadowRoot` eventually calls `createShadowRootPolyfill`.  So I removed the css isolation call, e.g:

```
// CSS isolation.
    installCssTransformer(shadowRoot, css => {
      return transformShadowCss(shadowRoot, css);
    });
```

inside `createShadowRootPolyfill`, and now css is only isolated when shadow css is not supported in `createShadowRoot`, therefore I can remove the check above.

Sorry for the lengthy answer, but took me a while to jog my memory on this haha!